run_app.py

```
@@ -1,7 +1,8 @@
 from streamlit.web import cli
+import sys
```

Ruff (0.8.2)

`sys` imported but unused

Remove unused import: `sys`

(F401)